### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by RoPP**
Les 4 premiers commits sont utilitaires, le dernier créé deux variables work_dir et project_dir, respectivement /data/work/{{ user }} et /data/project/{{ user }}.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (jocelynj) [inline .editorconfig]: À quoi sert ce fichier ?

File: init.fish [hunk 2]
```diff
@@ -0,0 +1,3 @@
+set -xg ANSIBLE_HOSTS (pwd)/hosts
+set -xg ANSIBLE_INVENTORY (pwd)/hosts
+set -xg ANSIBLE_TRANSPORT "ssh"
```
Comment (jocelynj) [inline init.fish]: I thought that fish used the same syntax than bash to export parameters.

In any case, I'm fine with this commit.

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+// Placez vos paramètres dans ce fichier pour remplacer les paramètres par défaut et les paramètres utilisateur.
+{
+    "files.associations": {
+        "*.yml": "ansible"
+    }
+}
```
Comment (jocelynj) [inline .vscode/settings.json]: vscode ? What is it used for ?

File: .editorconfig (hunk 1 above)
Comment (RoPP) [inline .editorconfig]: Almost all, natively or with a plugin: http://editorconfig.org/#download
The goal of this config file is to define a consistent (basic) coding style between all developers.

File: .vscode/settings.json (hunk 3 above)
Comment (RoPP) [inline .vscode/settings.json]: Visual Studio Code, the open source Microsoft editor, not that bad :)
This commit can be rejected if you don't want to track editor specific config file (maybe just add .vscode in .gitignore then).

**Pull request merged**